src/latticpp/ckks/keygen.cpp

```diff
@@ -11,10 +11,55 @@ namespace latticpp {
         return KeyGenerator(lattigo_newKeyGenerator(params.getRawHandle()));
     }
 
+    SwitchingKey getRotationKey(const RotationKeys &rotationKeys, uint64_t galEl) {
+      return SwitchingKey(lattigo_getRotationKey(rotationKeys.getRawHandle(), galEl));
+    }
+
+    void setRotationKey(const RotationKeys &rotKeys, const SwitchingKey &rotKey, uint64_t galEl) {
+      lattigo_setRotationKey(rotKeys.getRawHandle(), rotKey.getRawHandle(), galEl);
+    }
+
+    uint64_t rotationKeyExist(const RotationKeys &rotationKeys, uint64_t galEl) {
```

Comment:
I guess this would technically be `switchingKeyExist`. The naming is confusing in Lattigo. I'm not sure if being consistent with their naming is the right thing to do. Perhaps we should just refer to switching keys in the context of a rotation key set as "rotation keys"? Happy to hear your thoughts.

Reply:
Yes, this confusion origins from the Lattigo library. I don't have a strong opinion, but i think we can keep the following naming convention: the set of rotation keys are called RotationKeys, but a single rotation key is called a SwitchingKey.